helm/charts/hpe-csi-driver/templates/hpe-csi-node.yaml

```diff
@@ -130,6 +142,10 @@ spec:
             - name: DISABLE_NODE_CONFIGURATION
               value: "true"
             {{- end }}
+            {{ if .Values.disableNodeMonitor -}}
+            - name: DISABLE_NODE_MONITOR
+              value: "true"
```

Comment:
The value is set false everywhere else

Reply:
The default behavior is to have the Node Monitor enabled, hence false. If a user elects to `disableNodeMonitor`, it's set to true.